tests/py_localhost/README.md

```diff
@@ -1,12 +1,36 @@
 # Local server
 
-A local server based on the [example](https://github.com/python-hyper/h2/blob/master/examples/asyncio/asyncio-server.py) from hyper h2 server. Minor changed based on the example to response the headers of requests back within the headers from `/echo`.
+Local server based on [python-hyper/h2](https://github.com/python-hyper/h2).
 
 ## How to run the server
 
-Python 3.5+ required.
+Python 3.5+ required. Command here works for OSX and linux.
```

Comment:
trivial: don't need this comment. I think it's understood that the exact command (python vs python3 vs python3.10 etc) might differ